projects/contributor-experience.md

```diff
@@ -0,0 +1,111 @@
+  responsibilities within the project (triagers, approvers, maintainers, committee
+  members), which leads to friction in understanding of the overall project goals,
+  to misalignment in implementations and the feeling of contributors being remote
+  to other parts of the project.
+* No explicit contributor guidebook that helps contributors to answer common
+  questions no matter where they are on the contributors' ladder.
+* Mentorship for new contributors or contributors stepping up into new roles to
+  make them part of the project and aware of.  
+* Only implicit and unstructured recognition for contributors, which would help
+  contributors to feel seen and recognized.
+* No structure collaboration across SIGs, especially in the case of cross-cutting
+  concerns (Security, Docs, End-User, ...)
+* SIGs are short of contributors and require some help to advertise their need
+  (blog post, social posts, etc) and to lower the barrier of entry into their
+  sub-project.
+* _add more_
+
+Note that there are individuals within the project who try to tackle some of those
+aspects as individuals and not in a team effort, notable:
+
+* [Add skeleton for maintainer and contributor guidebooks](https://github.com/open-telemetry/community/pull/2051)
+* Mentorship efforts, e.g. [Outreachy](https://cloud-native.slack.com/archives/C060GFUL0P6)
+* Surveys by the End User SIG that aim towards contributor experience
+* _add more_
+
+Also related to this initiative is the [Check-In process of the Governance Committee](https://github.com/open-telemetry/community/blob/main/gc-check-ins.md)
+
+### Goals, objectives, and requirements
+
+The goal of this project is to establish a cross-cutting "SIG Contributor Experience",
+which is "responsible for improving the experience of those who upstream contribute to
+the OpenTelemetry project. [They] do this by creating, and maintaining programs and processes that promote community health and reduce project friction, while retiring those programs and processes that don't. Being conscientious of our contributor base is critical to scaling the project, growing the ecosystem, and helping the project succeed". (via [K8s Contributor Experience Special Interest Group Charter](https://github.com/kubernetes/community/blob/master/sig-contributor-experience/charter.md))
+
+While this is "yet another SIG" initiative, we believe that **not** establishing
+such a SIG is not an option: the problems outlined above will get bigger not
+smaller and while the final ownership of the health of the project is with the GC
+this requires a coordinated effort and more hands across the project to be addressed
+successfully.
+
+Note that similar to other cross-cutting SIGs (Docs, Security, End-User) this SIG will not exclusively own
+the domain of contributor experience, but they take responsibility to drive efforts in close collaboration
+with SIGs and Committees.
+
+## Deliverables
```

Comment:
Not sure if it needs explicit mention in the list of deliverables, but as I have discussed with some of you IMO an interesting area to explore here is to have more blog posts on the style of  "here's what's happening under the hood" or "we had this interesting technical problem which we fixed in this way" from SIGs. [The "Inside Rust" blog](https://blog.rust-lang.org/inside-rust/index.html) is a good example of the type of content that I think would be beneficial here.

There are other benefits, but in terms of Contributor Experience:
1. It can serve as recognition for contributors (we recognize you worked on this and you get to share with the community and the wider world
2. It can get more people interested and excited about development within OpenTelemetry (most content right now is focused on attracting more _users_ to OpenTelemetry, not so much developers)

Reply:
Great idea, I'll add that via https://github.com/open-telemetry/community/pull/2162/files#r1646316497